packages/components/src/navigable-container/test/tababble-container.tsx

```diff
@@ -51,7 +55,12 @@ describe( 'TabbableContainer', () => {
 
 		const onNavigateSpy = jest.fn();
 
-		render( <TabbableContainerTestCase onNavigate={ onNavigateSpy } /> );
+		render(
+			<TabbableContainerTestCase
+				onNavigate={ onNavigateSpy }
+				eventToOffset={ stubEventToOffset }
```

Comment:
`eventToOffset` is [a required prop](https://github.com/WordPress/gutenberg/blob/75ee44cc372cd58627454bcb44545b6929f01bb2/packages/components/src/navigable-container/container.tsx#L103-L108) of  `TabbableContainer`.

But maybe you'd like it to be optional.

Reply:
>Based on what context did you mark it as required?

My mistake, `eventToOffset` actually shouldn't be a required prop of `TabbableContainer`. Reverted in https://github.com/WordPress/gutenberg/pull/49377/commits/2d4ea7e27cb244f14cc9c91cfd346bbaff0bd5d6.

I confused [TabbableContainer](https://github.com/WordPress/gutenberg/blob/de067f252f8a4748dc147b1a5fe476a1dfbe16e6/packages/components/src/navigable-container/tabbable.ts) with [container.ts](https://github.com/WordPress/gutenberg/blob/de067f252f8a4748dc147b1a5fe476a1dfbe16e6/packages/components/src/navigable-container/container.js).

>Looking at how the component is used across tests / stories / gutenberg repo, is the props always defined ?

1. Tests: No, there [was no](https://github.com/WordPress/gutenberg/blob/de067f252f8a4748dc147b1a5fe476a1dfbe16e6/packages/components/src/navigable-container/test/tababble-container.js) `eventToOffset` prop in the test of `container.js` before this PR.
2. Stories: No
3. Gutenberg repo: No

Reply:
Makes sense, thank you for double checking!